server/game/game.js

```diff
@@ -766,9 +766,17 @@ class Game extends EventEmitter {
         }
     }
 
+    checkForTimeExpired() {
+        if(this.timeLimit.isTimeLimitReached && !this.finishedAt) {
+            this.addAlert('success', 'The game has ended because the timer has expired.  Timed wins are not currently implemented');
+            this.finishedAt = new Date();
```

Comment:
Elsewhere, we record a `winReason`. Should we record one here as `'timer'`, even if we make no determination about the winner?

Reply:
I was thinking about that, but wasn’t sure I want to record the result yet until the timed win stuff is added(and there’s no point setting winreason if the result isn’t recorded)